### ISSUE TITLE
Set `pause_on_next_bootstrap` on an offline cluster

### ISSUE THREAD

**Issue by kwannoel** [risingwavelabs/risingwave#17904]
Support changing it in `risingwave.toml`. In certain cases the cluster is crash looping, and because in standalone modes the faults between different nodes are not isolated as they are all embedded, the cluster can't be easily recovered.

Edit: Workaround for now: `RW_COMPUTE_NODE_ROLE=serving`. This pauses all streaming jobs.

**Comment by BugenZhao** [comment]
> In certain cases the cluster is crash looping

I suppose you are referring to process panicking. Regular soft failures in streaming jobs should not affect the capability of setting system parameters with SQL interfaces.

**Comment by kwannoel** [comment]
> > In certain cases the cluster is crash looping
> 
> I suppose you are referring to process panicking. Regular soft failures in streaming jobs should not affect the capability of setting system parameters with SQL interfaces.

Yeah panic, regular soft failure will just trigger recovery, and meta + frontend should still be responsive.

**Comment by kwannoel** [comment]
Already supported.
```
[system]
pause_on_next_bootstrap = true
```

**Comment by BugenZhao** [comment]
> Already supported.
> 
> ```
> [system]
> pause_on_next_bootstrap = true
> ```

Ah. Specifying system parameters through config file may only take effect on fresh cluster.

**Comment by kwannoel** [comment]
@EdwinaZhu will be working on this, by providing a new `pause_on_next_boostrap_offline` configuration to the meta node.
